packages/examples/src/ComponentExamples.test.tsx

```diff
@@ -94,6 +94,9 @@ function setCacheFullWarning(enabled: boolean) {
   };
 }
 
+/** Run tests on these packages' examples */
+const includedPackages = ['date-time', 'office-ui-fabric-react', 'react-cards', 'react-focus', 'react-button'];
```

Comment:
This list is probably out of date. Also some of the changes here may still be buggy.